src/game/client/draw/drawable.cpp

```diff
@@ -1577,7 +1577,7 @@ void Drawable::Xfer_Snapshot(Xfer *xfer)
     if (m_object != nullptr) {
         object_id = m_object->Get_ID();
     } else {
-        object_id = OBJECT_UNK;
+        object_id = INVALID_ID;
```

Comment:
INVALID_ID reads ambiguous. Perhaps call this INVALID_OBJECT_ID?

Reply:
well indeed its INVALID_ID
BFME1 also shows it

Reply:
while probably should go with INVALID_OBJECT_ID for consistency with others

Reply:
Fixed.